core/models.py

```diff
@@ -72,11 +72,11 @@ class Candidate(models.Model):
     name = models.CharField(max_length=100, blank=False)
     std_no_regex = RegexValidator(regex=r"^\d{7}$", message="Invalid Student Number", code="400")
     std_no = models.CharField(validators=[std_no_regex], blank=True, max_length=7, null=True)
-    university_roll_no = models.CharField(max_length=10, blank=True, unique=True, null=True)
-    email = models.EmailField(unique=True, blank=False)
+    university_roll_no = models.CharField(max_length=10, blank=False, unique=True, null=True)
```

Comment:
university_roll_no :  blank=False & null= true. You find that right ? 

& If you still accept null  in Django model then a null string in Django model can have any of two value i.e either NULL or Empty String. So two NULL or two Empty String as university_roll_no  violates **unique=True** .

Reply:
sir will fix this
thanx